HW_7/logger.py

```diff
@@ -0,0 +1,11 @@
+from datetime import datetime
+
+
+def log(func):
+    def wapper(*args):
+        with open('log.csv', 'a', encoding='utf-8') as log_file:
+            log_file.write(f'{datetime.now()} - функция запущена' 
+                            '{func.__name__} {"с аргументами" if args else "без аргументов"}' 
+                            '({func.__doc__})\n')

```

Comment:
Если вы так переносите форматированные строки, то нужно перед каждой новой строкой ставить букву `f`.
```suggestion
                            f'{func.__name__} {f"с аргументами: {args}" if args else "без аргументов"}' 
                            f'({func.__doc__})\n')
```
Иначе ваш логгер будет выглядеть так
![image](https://user-images.githubusercontent.com/49741194/208607877-08c41475-619f-4d83-a2af-1462dab10a7c.png).